src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -14,22 +14,12 @@
 use Symfony\Component\Console\Helper\ProgressBar;
 use Symfony\Component\Console\Helper\Helper;
 use Symfony\Component\Console\Output\StreamOutput;
-use Symfony\Component\Console\Tests;
-
-require_once __DIR__.'/../ClockMock.php';
 
+/**
+ * @group clockMock
```

Comment:
I think I don't like having mock in there.

I do understand that using a group is much easier (I recently tried to add an annotation for BF :)), but that does not make it better (and not sure it makes sense to filter those tests).

Time sensitive sounds much better.